constants/type_constants.asm

```diff
@@ -19,15 +19,7 @@ PHYSICAL EQU const_value
 	const STEEL
 
 UNUSED_TYPES EQU const_value
-	const TYPE_10
-	const TYPE_11
-	const TYPE_12
-	const TYPE_13
-	const TYPE_14
-	const TYPE_15
-	const TYPE_16
-	const TYPE_17
-	const TYPE_18
+	const_skip 9
```

Comment:
Super minor nitpick by why not `const_next 19` here? It's not like the data for any of these types is defined anywhere, and kinda clear they just decided to start at 20 for the special types.